src/main/java/org/synyx/urlaubsverwaltung/sickdays/web/SickDaysOverviewViewController.java

```diff
@@ -72,7 +73,7 @@ public String filterSickNotes(@ModelAttribute("period") FilterPeriod period) {
     public String periodsSickNotes(@RequestParam(value = "from", required = false) String from,
```

Comment:
muss nu required sein?

Reply:
Also es wird durch die `Asserts` in `FilterPeriod` abgefangen und entsprechend eine `IllegalArgumentException` geworfen. Führt dann aber zu einem unschönen `500`er sollte ja eher `400` sein das könnte man durch required hinbekommen.